core/src/test/java/com/sequenceiq/cloudbreak/core/bootstrap/service/host/ClusterHostServiceRunnerTest.java

```diff
@@ -206,24 +209,24 @@
     @Mock
     private CsdParcelDecorator csdParcelDecorator;
 
+    @Mock
+    private LoadBalancerSANProvider loadBalancerSANProvider;
+
     @InjectMocks
     private ClusterHostServiceRunner underTest;
 
-    @Before
+    @BeforeEach
     public void setUp() {
-        initMocks(this);
-        when(stack.getEnvironmentCrn()).thenReturn("envCrn");
-        when(environmentConfigProvider.getParentEnvironmentCrn(any())).thenReturn("envCrn");
+        lenient().when(stack.getEnvironmentCrn()).thenReturn("envCrn");
+        lenient().when(environmentConfigProvider.getParentEnvironmentCrn(any())).thenReturn("envCrn");
     }
 
     @Test
     public void shouldUsecollectAndCheckReachableNodes() throws NodesUnreachableException {
         try {
-            expectedException.expect(NullPointerException.class);
             underTest.runClusterServices(stack, cluster, Map.of());
-        } catch (Exception e) {
+        } catch (NullPointerException e) {
```

Comment:
Do we really expect an NPE from the test code? Kindly note that this test might pass even if the code throws or doesn't throw an NPE.

Reply:
you're right, fixed it, slipped through the refactor